Supporting Documents/BS_SD.adoc

```diff
@@ -1199,13 +1219,15 @@ If the developer specifies 0.01% FMR and 1% FNMR in FIA_MBV_EXT.1, at least 30,0
 
 [[qualityassessmentcriteria]]
 == Developer’s quality assessment criteria report of biometric samples
-The TOE shall only use biometric samples of sufficient quality as required by FIA_MBE_EXT.2 and FIA_MBV_EXT.2. This means that the TOE shall assess the quality of samples based on quality assessment criteria, namely, quality scores of samples measured by quality metrics and defined thresholds to reject low-quality samples and the assessment criteria shall contribute to the TOE’s performance, especially FRR and FNMR. 
+The term “quality” is used in FIA_MBE_EXT.2 and FIA_MBV_EXT.2 and these SFRs require the TOE to use biometric samples of sufficient quality. However, the quality of a biometric sample is interpreted differently throughout literature. In general, quality is defined as an indicator of the usefulness of the biometric sample for biometric enrolment and verification. The TOE can use this indicator to improve the TOE’s performance, especially FRR and FNMR, because the TOE can reject “bad” or low-quality samples at an early phase that would cause performance degradation. 
```

Comment:
Equivalent.

```suggestion
The term “quality” is used in FIA_MBE_EXT.2 and FIA_MBV_EXT.2 and these SFRs require the TOE to use biometric samples of sufficient quality. However, the quality of a biometric sample is interpreted differently throughout literature. In general, quality is defined as an indicator of the usefulness of the biometric sample for biometric enrolment and verification. The TOE can use this indicator to improve the TOE’s performance, especially FNMR, because the TOE can reject “bad” or low-quality samples at an early phase that would cause performance degradation. 
```